config/config.exs

```diff
@@ -26,6 +26,14 @@ config :logger, :console,
 # Use Jason for JSON parsing in Phoenix
 config :phoenix, :json_library, Jason
 
+#esbuild config 
+config :esbuild,
+  version: "0.12.18",
```

Comment:
Use v0.14 and above for esbuild, v0.12.18 is too behind https://www.npmjs.com/package/esbuild